optimade/server/main.py

```diff
@@ -118,21 +122,54 @@ def get_structures(request: Request, params: EntryListingQueryParams = Depends()
     "/info",
     response_model=Union[InfoResponse, ErrorResponse],
     response_model_skip_defaults=True,
-    tags=["Structure"],
+    tags=["Info"],
 )
-def get_info(request: Request, params: EntryListingQueryParams = Depends()):
+def get_info(request: Request):
     return InfoResponse(
         meta=meta_values(str(request.url), 1, more_data_available=False),
         data=BaseInfoResource(
             attributes=BaseInfoAttributes(
                 api_version="v0.9",
                 available_api_versions={"v0.9": "http://localhost:5000/"},
-                entry_types_by_format={"json": ["structures"]},
+                entry_types_by_format={"jsonapi": ["structures"]},
             )
         ),
     )
 
 
+@app.get(
+    "/structures/info",
+    response_model=Union[EntryInfoResponse, ErrorResponse],
+    response_model_skip_defaults=True,
+    tags=["Structure", "Info"],
+)
+def get_structures_info(request: Request):
+    return EntryInfoResponse(
+        meta=meta_values(str(request.url), 1, more_data_available=False),
+        data=EntryInfoResource(
+            id="",
+            type="structures/info",
+            attributes=EntryInfoAttributes(
+                description="attributes that can be queried",
+                properties={
+                    "exmpl_p": EntryPropertyInfo(description="a sample custom property")
+                },
+                output_fields_by_format={
```

Comment:
Should we generate this list programmatically from the properties in `StructureResourceAttributes` (see e.g. #45 and https://github.com/Materials-Consortia/optimade-python-tools/blob/63a8bb209e5bd9ffc33ededb8135cee31b3f140f/optimade/server/models/structures.py#L6-L12)? The same would apply to other endpoints.

Reply:
@ml-evs yes, we should do this, but we can in a separate PR. I'll merge this.